File: src/main/java/com/my/admin/app/commerce/model/api/response/CashStatusResponse.java
package com.my.admin.app.commerce.model.api.response;

import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

@Data
public class CashStatusResponse {

    @ApiModelProperty(value = "Cash Status Id", example = "1")
    private String id;

    @ApiModelProperty(value = "Cash Status name", example = "Activa")
    private String name;
}

File: src/main/kotlin/com/bk/reactive/app/commerce/admin/my/model/api/response/CashStatusResponse.kt
package com.bk.reactive.app.commerce.admin.my.model.api.response

import com.fasterxml.jackson.annotation.JsonInclude
import io.swagger.annotations.ApiModelProperty

@JsonInclude(JsonInclude.Include.NON_EMPTY)
class CashStatusResponse(
    @ApiModelProperty(value = "Cash Status Id", example = "1")
    val id: String,

    @ApiModelProperty(value = "Cash Status name", example = "Activa")
    val name: String
)